apps/train_loc/test/train_loc/manager_test.exs

```diff
@@ -187,4 +187,107 @@ defmodule TrainLoc.ManagerTest do
   def send_self(message) do
     send(self(), message)
   end
+
+  describe "generate_feed/2" do
+    test "generates a feed from new vehicles" do
+      vehicles = [
+        %Vehicle{
+          heading: 0.0,
+          latitude: 42.240323,
+          longitude: -71.128225,
+          speed: 0,
+          trip: :unassigned,
+          timestamp: ~U[2022-01-20 23:43:42Z],
+          vehicle_id: 1506
+        },
+        %Vehicle{
+          heading: 0.0,
+          latitude: 42.240323,
+          longitude: -71.127625,
+          speed: 0,
+          trip: :unassigned,
+          timestamp: ~U[2022-01-20 23:43:42Z],
+          vehicle_id: 1507
+        },
+        %Vehicle{
+          heading: 199.0,
+          latitude: 42.23879,
+          longitude: -71.13356,
+          speed: 1,
+          timestamp: ~U[2022-01-20 23:43:42Z],
+          trip: "745",
+          vehicle_id: 1823
+        },
+        %Vehicle{
+          heading: 123.4,
+          latitude: 56.78901,
+          longitude: -23.45678,
+          speed: 1,
+          timestamp: ~U[2022-01-20 23:43:42Z],
+          trip: "123",
+          vehicle_id: 1234
+        },
+        %Vehicle{
+          heading: 120.4,
+          latitude: 52.78901,
+          longitude: -24.45678,
+          speed: 1,
+          timestamp: ~U[2022-01-20 23:11:42Z],
+          trip: "129",
+          vehicle_id: 56_789
+        }
+      ]
+
+      assert [
+               %{
+                 "id" => "54134678",
+                 "vehicle" => %{
+                   "position" => %{
+                     "bearing" => 0.0,
+                     "latitude" => 42.240323,
+                     "longitude" => -71.128225,
+                     "speed" => 0.0
+                   },
+                   "timestamp" => 1_642_722_222,
+                   "trip" => %{"start_date" => "20220120"},
```

Comment:
question: is this valid without a `trip_short_name`?

Reply:
Well, that's the current behavior for unassigned vehicles: https://github.com/mbta/commuter_rail_boarding/blob/213eb4ac72e5c678b06f3298e98c36b9a9dbd1ff/apps/train_loc/test/train_loc/encoder/vehicle_positions_enhanced_test.exs#L74-L98

Reply:
`trip_short_name` doesn't appear to be part of the specification, either: https://developers.google.com/transit/gtfs-realtime/reference#message-tripdescriptor